lib/middleware/route.js

```diff
@@ -69,13 +69,18 @@ exports.middleware = function route(next, app) {
     function addRoute(method, path, fn, name) {
         var keys = [];
         var spec = {keys: keys, fn: fn};
-        spec.pattern = path instanceof RegExp ? path : normalizePath(path, keys);
-        spec.weight = calcWeight(path);
+        if (path instanceof RegExp) {
+        	spec.pattern = path;
```

Comment:
I think there is a tab instead of spaces. Minor thing, but cosmetics & consistency.